src/utils.js

```diff
@@ -38,7 +38,34 @@ export function timeAfter(seconds) {
     return currentTime() + seconds;
 }
 
-export const TEST_VIDEO_ID = '495nCzxM9PI';
+// https://gist.github.com/creationix/7435851
+// Joins path segments.  Preserves initial "/" and resolves ".." and "."
+// Does not support using ".." to go above/outside the root.
+// This means that join("foo", "../../bar") will not resolve to "../bar"
+export function pathJoin(...args) {
```

Comment:
keep utils in their own paths. E.g. `src/utils/joinPath.js`. Easier to traverse and find things.